examples/public-zone/variables.tf

```diff
@@ -42,3 +42,9 @@ variable "labels" {
     version = "bar"
   }
 }
+
+variable "enable_logging" {
```

Comment:
nit: lets not add this here and just hard code in example if we want to test this field. We are trying to move towards having most inputs to the module inlined within the example.

Reply:
done.